lib/cosmos/gui/qt_tool.rb

```diff
@@ -45,10 +45,37 @@ def initialize(options)
       @options = options
       @about_string = nil
 
+      base_class = self.class.to_s.split('::')[-1].downcase
+      config_dir = File.join(Cosmos::USERPATH, 'config', 'tools', base_class)
+      if File.exist? config_dir
+        @options.config_dir = config_dir
+        options.config_file = verify_config(@options.config_file, ".txt", base_class)
```

Comment:
You are using options and @options inconsistently here.   It works because they are the same object, but be consistent.